web/src/components/molecules/MyIntegrations/WebhookList/WebhookCard/index.tsx

```diff
@@ -1,29 +1,61 @@
 import { Switch } from "antd";
+import { useCallback } from "react";
 
 import Card from "@reearth-cms/components/atoms/Card";
 import Icon from "@reearth-cms/components/atoms/Icon";
+import { Webhook, WebhookTrigger } from "@reearth-cms/components/molecules/MyIntegrations/types";
 
 export type Props = {
-  title: string;
-  url: string;
+  webhook: Webhook;
+  onWebhookDelete: (webhookId: string) => Promise<void>;
+  onWebhookUpdate: (data: {
+    webhookId: string;
+    name: string;
+    url: string;
+    active: boolean;
+    trigger: WebhookTrigger;
+  }) => Promise<void>;
+  onWebhookSettings: (webhookId: string) => void;
 };
 
-const WebhookCard: React.FC<Props> = ({ title, url }) => {
+const WebhookCard: React.FC<Props> = ({
+  webhook,
+  onWebhookDelete,
+  onWebhookUpdate,
+  onWebhookSettings,
+}) => {
+  const handleWebhookDelete = useCallback(() => {
+    onWebhookDelete(webhook.id);
+  }, [onWebhookDelete, webhook.id]);
+
+  const handleWebhookUpdate = useCallback(
+    (active: boolean) => {
+      onWebhookUpdate({ ...webhook, active, webhookId: webhook.id });
+    },
+    [onWebhookUpdate, webhook],
+  );
+
   return (
     <Card
       style={{ marginTop: 16 }}
       title={
         <>
-          {title} <Switch checkedChildren="ON" unCheckedChildren="OFF" defaultChecked />
+          {webhook.name}{" "}
```

Comment:
```suggestion
          <span style={{display: "inline-block", marginLeft:"8px"}}>{webhook.name}</span>
```
Is this `{" "}` for adding a space between `weebhook.name` and `<Switch />`
if so please consider this suggestion, otherwise please delete `{" "}`

Reply:
Fixed.
https://github.com/reearth/reearth-cms/blob/0d6f8498a206da71b1c60aa624689d49b4bd8ab3/web/src/components/molecules/MyIntegrations/WebhookList/WebhookCard/index.tsx#L44